lib/main.coffee

```diff
@@ -2,13 +2,12 @@
 module.exports =
   instance: null
   config:
-    lintOnFly:
-      description: 'Lint files while typing, without the need to save them'
+    lintAsYouType:
       type: 'boolean'
       default: true
       order: 1
-    lintOnFlyInterval:
-      description: 'Interval at which providers are triggered on the fly'
+    lintAsYouTypeInterval:
```

Comment:
Please do not rename this, it will create issue with people's already existing settings. Rather you can change the title by adding a `name` field to this config option.